tests/phpunit/tests/rest-api/class-wp-rest-block-directory-controller-test.php

```diff
@@ -0,0 +1,194 @@
+<?php
+/**
+ * Test WP_REST_Block_Directory_Controller_Test()
+ *
+ * @package Gutenberg
+ * phpcs:disable
+ */
+class WP_REST_Block_Directory_Controller_Test extends WP_UnitTestCase {
+	protected $controller = null;
+
+	function setUp() {
+		parent::setUp();
+
+		global $wp_rest_server;
+		$wp_rest_server = new \WP_REST_Server;
+		$this->server   = $wp_rest_server;
```

Comment:
We need the `protected $server;` definition for the server.

Reply:
I'm not sure it's needed at all, since I've switched the test functions to use `rest_get_server()->dispatch()`